tools/build/scripts/rush/audit.js

```diff
@@ -53,6 +53,7 @@ const rushCommonDir = path.join(__dirname, "../../../../common/");
     "GHSA-76p3-8jx3-jpfq", // https://github.com/advisories/GHSA-76p3-8jx3-jpfq @bentley/react-scripts>loader-utils
     "GHSA-3rfm-jhwj-7488", // https://github.com/advisories/GHSA-3rfm-jhwj-7488 @bentley/react-scripts>react-dev-utils>loader-utils
     "GHSA-hhq3-ff78-jv3g", // https://github.com/advisories/GHSA-hhq3-ff78-jv3g @bentley/react-scripts>react-dev-utils>loader-utils
+    "GHSA-27h2-hvpr-p74q", // https://github.com/advisories/GHSA-27h2-hvpr-p74q @bentley/core-backend>azurite>jsonwebtoken
```

Comment:
Do you know if an issue has been filed with azurite and if they intend to do anything about it? They're still on jsonwebtoken  8.5.1.

Reply:
I couldn't find any issues or pull requests in the azurite repo around that package. I did make sure that this package was not being used in any non dev dependencies before adding the exception though.